packages/fuselage/src/components/Message/MessageEmoji.tsx

```diff
@@ -0,0 +1,33 @@
+import type { ReactNode } from 'react';
+import React from 'react';
+
+import { MessageEmojiBase } from './MessageEmojiBase';
+
+type MessageEmojiProps = {
+  name: string;
+  className?: string;
+  image?: string;
+  big?: boolean;
+  children?: ReactNode;
+};
+
+export const MessageEmoji = ({
+  name,
+  className,
+  image,
+  big,
+  children,
+}: MessageEmojiProps) => (
+  <MessageEmojiBase
+    className={[
+      'rcx-message__emoji',
+      className,
```

Comment:
```suggestion
      className && 'rcx-message__emoji',
      className,
```
Can we add a conditional here? cause when the className is undefined we will render the emoji text like `:random-text:` so we don't need the emoji style in this scenario.

Reply:
also we need the `name` in the class name to render the correct style of `emoji`

Reply:
I kind of agree with the first comment. I know we need for this to be "sizeless" whenever the emoji doesn't exist, but not sure if we should depend on the className. For ex: if for some reason we decide to pass another className, this implementation falls apart. Maybe we need an aditional prop?

On the second one, I don't se a reason to add the name in the className, because that is Application specific. If the application needs to send the name as a className, it can do so by including it in the className prop.

Reply:
I just realized that for the first comment, we can just not use this component :p. I'll leave it like it is...